packages/synthetic-chain/public/upgrade-test-scripts/env_setup.sh

```diff
@@ -80,6 +83,20 @@ await_agd_startable() {
   fi
 }
 
+save_latest_node_info() {
+  PID="$(cat "$PID_FILE")"
+
+  if ps --pid "$PID" > /dev/null
+  then
+    NODE_STATUS="$(agd status)"
+    BLOCK_HEIGHT="$(echo "$NODE_STATUS" | jq '.SyncInfo.latest_block_height' --raw-output)"
+    echo "Saving node status at block height $BLOCK_HEIGHT"
+    echo "$NODE_STATUS" > "$STATUS_FILE"
```

Comment:
So this will be needed to capture the height from which we gonna start a3p follower with state-sync, right?

Reply:
Yes